build_deps.sh

```diff
@@ -5,5 +5,7 @@ BUILDDIR="$(readlink -f $BUILDDIR)/build"
 URIDIR="$BUILDDIR/uriparser"
 
 if [ -n "$URIDIR/lib/liburiparser.a" ]; then
+    cd deps/ngx_url_parser && make static-lib
```

Comment:
I wouldn't keep source files along with builded lib, what about building to the build dir, same as we do for uriparser?